File: SmartCredentials_aOS/oneclickbusinessclient/src/main/java/com/example/oneclickbusinessclient/rest/RetrofitClient.java
package com.example.oneclickbusinessclient.rest;

public class RetrofitClient extends BaseRetrofitClient {

    private static final String RECOMMENDER_URL = "http://lbl-recommender.superdtaglb.cf/debug/";

    @Override
    public String getRecommenderUrl() {
        return RECOMMENDER_URL;
    }
}

File: samples/hackathon/app/src/main/java/com/operatortokenocb/network/RetrofitClient.kt
package com.operatortokenocb.network

class RetrofitClient : BaseRetrofitClient() {

    companion object {
        private const val RECOMMENDER_URL = "http://lbl-recommender.superdtaglb.cf/debug/"
    }

    override fun recommenderUrl(): String {
        return RECOMMENDER_URL
    }
}
